### ISSUE TITLE
Watched unexpected filepath

### ISSUE THREAD

**Issue by pws019** [floatdrop/gulp-watch#243]
Globs: `["E:\\zhsj\\elasticforce\\config/../webapp/src/"]`
Filepath: `E:\zhsj\elasticforce\webapp\src\assets\css\a.css`
Options:
```js
{
  "events": [
    "add",
    "change",
    "unlink"
  ],
  "ignoreInitial": true,
  "readDelay": 10
}
```

**Comment by UltCombo** [comment]
Thanks for reporting. Looks like we are missing a `path.normalize` or `path.resolve` somewhere. It should be fairly straightforward to fix, PRs are welcome. I should be able to take a look tonight if no one else gets to it until then.

**Comment by UltCombo** [comment]
Hey, sorry for the delay.

I've added several tests and could not reproduce the issue (#247).
However, I believe I have isolated the only possible cause for this issue and patched it here: #248

Can you please try the latest version (v4.3.9) to check if the issue has been fixed?

---

As a side note, I've drastically improved the error logging so it should be far easier to debug these issues now.

Also, in case you still have the gulpfile that reproduces the problem, it would be nice if you could post it here so that we can add a proper regression test.

**Comment by UltCombo** [comment]
Closing as this is likely fixed in the latest version.

Repro code is still welcome if anyone can still reproduce the problem.
